libfuse/folderlist.go

```diff
@@ -95,6 +95,20 @@ func (fl *FolderList) addToFavorite(ctx context.Context, h *libkbfs.TlfHandle) (
 	return nil
 }
 
+// Create implements the fs.NodeCreater interface for FolderList.
+func (fl *FolderList) Create(ctx context.Context, req *fuse.CreateRequest, resp *fuse.CreateResponse) (_ fs.Node, _ fs.Handle, err error) {
+	fl.fs.log.CDebugf(ctx, "FL Create")
+	defer func() { fl.reportErr(ctx, libkbfs.WriteMode, "", err) }()
+	return nil, nil, fl.fs.writeAccessError(ctx, nil, libkbfs.BuildCanonicalPath(fl.public, libkbfs.CanonicalTlfName(req.Name)))
```

Comment:
To me I think it makes sense to return permission denied (EACCES) if a program (or user) tries to create /keybase/foo.txt or /keybase/private/foo.txt ... It seems more appropriate than ENOENT which implies not found?